### DISCUSSION TITLE
Two ptms on the same aa?

### DISCUSSION THREAD

**Discussion by BenoitBe** [vdemichev/DiaNN#1271] [General]
Dear Vadim,

I am not sure, but can DIA-NN support two ptms in the same aa? Or do I have to add specific information in command prompt?

Thanks in advance,

Benoit

**Comment by vdemichev** [comment]
Hi Benoit,

Yes, but they will be mutually exclusive (i.e. not occur at the same time). That is if you want two at the same time, just specify a PTM that is a combination of the two.

Best,
Vadim

**Reply by BenoitBe** [reply]
Ok thx Vadim, I had the same idea for occur at the same time :)
